docs/algorithms/theory-idars.rst

```diff
@@ -0,0 +1,114 @@
+IDaRS Theory
+====================
+
+IDaRS stands for **I**\ terative **D**\ raw **a**\ nd **R**\ ank **S**\ ampling,
+an algorithm introduced in an article
+`[Read/download]
```

Comment:
```suggestion
DOI: `[https://doi.org/10.1016/S2589-7500(21)00180-1]
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/383/commits/86d457bcf3485cfd7b4e0686a8f97ece71cf6e99